src/modules/admin/ui/components/columns.tsx

```
@@ -0,0 +1,114 @@
+  DropdownMenuTrigger,
+} from "@/components/ui/dropdown-menu";
+
+export const columns: ColumnDef<MemberGetOne>[] = [
+  {
+    accessorKey: "status",
+    header: "Status",
+    cell: ({ row }) => {
+      return (
+        <span
+          className={
+            true ? "font-medium text-green-600" : "font-medium text-red-600"
```

Biome (2.1.2)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)